src/renderer/components/swap/CurrencyInfo/CurrrencyInfo.component.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+import { Asset } from '@thorchain/asgardex-util'
+import BigNumber from 'bignumber.js'
+
+import * as Styled from './CurrencyInfo.styles'
+
+type CurrencyInfo = {
+  from?: { asset: Asset; priceRune: BigNumber }
+  to?: { asset: Asset; priceRune: BigNumber }
+}
+
+export const CurrencyInfo = ({ from, to }: CurrencyInfo) => {
+  if (!from || !to) {
+    return null
+  }
+  return (
+    <Styled.Container>
+      <div>
+        1 {from.asset.ticker} = {from.priceRune.dividedBy(to.priceRune).toFormat(5).toString()} {to.asset.ticker}
```

Comment:
u r absolutely right but IMO dot-notation chaining is more readable here then 
```ts
formatBn(from.priceRune.dividedBy(to.priceRune), 5).toString()
```
just because we need to 'parse' all of that braces when read the code. what do you think ?

Reply:
`formatBn(from.priceRune.dividedBy(to.priceRune), 5)` - no `toString()` is needed. 

I would prefer to use our helper functions we use everywhere. These are tested and we know what they do. If you think a `dot-notation chaining` is not readable with these functions, why not just avoid chaining here by extracting the result of the "chain" into a variable to put it into `formatBN`?

Reply:
oh. my bad here. we do not need `toString` there. thats true